0076-Hold_Rewards_for_Denylist_Hotspots.md

```diff
@@ -0,0 +1,22 @@
+Helium Improvement Proposal (HIP) Title: Temporarily Holding Rewards for Denied List Hotspots
+
+Authors:
+- Rob#8633
+- H3ct0R
+- ChatGPT
+
+## Introduction:
+The Helium network is a decentralized network that relies on hotspots to create and maintain the network. However, in some cases, the rewards of some hotspots may be denied due to potential fraudulent activity. While it is important to prevent fraud, it may also result in the unfair denial of rewards to legitimate hotspots.
+
+## Proposal:
+To address this issue, this HIP proposes a temporary holding of rewards for hotspots that have been placed on the denied list. The rewards would be held until a review and appeal process is completed. If the hotspot is deemed to be legitimate, the rewards would be returned to the hotspot owner.
+
+## Implementation:
+1. A review and appeal process will be established to allow hotspot owners to challenge their denied rewards.
+2. The rewards for denied hotspots will be temporarily held in a secure escrow account.
```

Comment:
Who will hold this account? Foundation? or the subDAO?

Reply:
I think it's too early to say, because we don't know how Oracles will work. But ideally it would be something automatically handled in some decentralise way so no one person can just run off with the HNT.

Reply:
I think it would be good to connect with some of the folks implementing oracles. Check this #core-dev or #blockchain channels in Discord and find a Core Dev who's working on it to talk to.